systers_portal/templates/account/signup.html

```diff
@@ -0,0 +1,63 @@
+{% extends "account/base.html" %}
+{% load url from future %}
+{% load i18n %}
+{% block head_title %}{% trans "Signup" %}{% endblock %}
+{% block content %}
+<head>
```

Comment:
Please never include CSS styling inside templates. This is bad practice. Once you forget where exactly you defined styling to a specific block, you will pull the hair from your head trying to find this unfortunate place.

We have "style.css" for that.